pages/index.vue

```diff
@@ -1,19 +1,94 @@
 <script setup lang="ts">
-import { useQuery } from '@vue/apollo-composable'
-import { GET_CHARACTERS } from '@/api/queries'
-import { ElTable, ElTableColumn, ElContainer } from 'element-plus'
+import { ElContainer, ElImage, ElPagination, ElCard } from "element-plus";
+import { useCharacterList } from "@/composables/useCharactersList";
+import BaseLoader from "@/base/BaseLoader";
```

Comment:
it could be auto imported 
https://v3.nuxtjs.org/examples/auto-imports/components